tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -141,9 +142,12 @@ struct btp_gap_start_discovery_cmd {
 #define BTP_GAP_STOP_DISCOVERY			0x0d
 
 #define BTP_GAP_CONNECT				0x0e
+#define BTP_GAP_CONNECT_LE			0x00
+#define BTP_GAP_CONNECT_BREDR		0x01
 struct btp_gap_connect_cmd {
 	bt_addr_le_t address;
 	uint8_t own_addr_type;
+    uint8_t transport;
```

Comment:
Formatting issue here: the indentation is different (spaces vs tab) from the rest of the struct